component/presentation.go

```diff
@@ -26,15 +26,15 @@ type RepoPresentation struct {
 }
 
 // Render renders the component.
-func (p *RepoPresentation) Render() *vecty.HTML {
+func (p *RepoPresentation) Render() vecty.ComponentOrHTML {
 	return elem.Div(
 		vecty.Markup(
-			prop.Class("list-entry go-package-update"),
+			vecty.Class("list-entry go-package-update"),
```

Comment:
This should be `vecty.Class("list-entry", "go-package-update")` to comply with the [vecty API](https://github.com/gopherjs/vecty/blob/master/doc/CHANGELOG.md#oct-14-2017-pr-155-major-breaking-change):

> Migrating users must use the new function and split their classes into separate strings, rather than a single space-separated string.

I can fix this.